Src/Campus.Infrastructure.Data/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Dapper" version="2.0.35" targetFramework="net48" />
```

Comment:
Change target to .net core 3.1 for all projects

Reply:
Changed